Research/ResearchUI/iOS/Step View Controllers/RSDActiveStepViewController.swift

```diff
@@ -77,11 +67,25 @@ open class RSDActiveStepViewController: RSDFullscreenImageStepViewController {
     
     /// A label that is updated to show a countdown. For example, "5:35".
     @IBOutlet open var countdownLabel: UILabel?
+    
+    /// A label that is displayed to the user when the countdown is finished.
+    @IBOutlet open var doneLabel: UILabel?
+    
+    /// Returns the allowed countdown units for the countdown label. Default is to show only seconds
+    /// if the duration is less than 90 seconds.
+    open var allowedCountdownUnits: NSCalendar.Unit {
+        if self.activeStep?.duration ?? 0 > 90 {
```

Comment:
I can never remember operator precedence for obscure-ish ones like ?? and >, so I'd like to see parentheses to make it explicit -- also makes it easier/faster for me to parse it even when I can eventually figure it out from context